CODEOWNERS

```diff
@@ -0,0 +1,2 @@
+# see https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners for more information
+Profiling/ @Unity-Technologies/multiplayer-tools
```

Comment:
why do we want this? 👀

Reply:
Because they should be added in PRs related to this code and thus you can use the codeowners file to encourage that

Reply:
so, before we merge this, should we gather current codeowners and put them here in this PR in one go as an initial start then?

Reply:
There is a difference between
Profiling/ 

and 

/Profiling/

Reply:
https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners

Sorry for not making it more clear